src/custom/pages/Profile/index.tsx

```diff
@@ -25,10 +25,27 @@ import { SupportedChainId as ChainId } from 'constants/chains'
 
 export default function Profile() {
   const referralLink = useReferralLink()
+  const { profileData, error } = useFetchProfile()
   const { account, chainId } = useActiveWeb3React()
-  const profileData = useFetchProfile()
   const lastUpdated = useTimeAgo(profileData?.lastUpdated)
 
+  const renderNotificationMessages = () => {
```

Comment:
I'm ok with the logic of showing one at a time, but I don't think you need the function here.
A simple const would suffice.